package.json

```
@@ -55,7 +55,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "nx format:write && git update-index --again",
+      "pre-commit": "nx format:write && bazel run //:buildifier && git update-index --again",
```

Can we use the bazel binaray from the node_modules as this would require installing bazel on the computer and we already have bazelisk in our node_modules.

maybe add npm script to package.json

```json
"bazel:format": "bazelisk run //:buildifier"
```

```suggestion
      "pre-commit": "nx format:write && npm run bazel:format && git update-index --again",
```